staging/src/k8s.io/kubectl/pkg/cmd/run/run.go

```diff
@@ -544,7 +544,7 @@ func getRestartPolicy(cmd *cobra.Command, interactive bool) (corev1.RestartPolic
 	case corev1.RestartPolicyNever:
 		return corev1.RestartPolicyNever, nil
 	}
-	return "", cmdutil.UsageErrorf(cmd, "invalid restart policy: %s", restart)
+	return "", cmdutil.UsageErrorf(cmd, "invalid restart policy: %s. Acceptable values are: %s, %s, %s.", restart, corev1.RestartPolicyAlways, corev1.RestartPolicyOnFailure, corev1.RestartPolicyNever)
```

Comment:
Looks good to me. Could you please add a test case for checking the error message in here https://github.com/kubernetes/kubernetes/blob/45836971f27ca70cd7742e8ee66e99e3c648cf9f/staging/src/k8s.io/kubectl/pkg/cmd/run/run_test.go#L17